### PR TITLE
Modal Featur

### PR DESCRIPTION
**Pull request by dhe19**
**Note:** I completed this project independently as my partner was unable to contribute due to external circumstances. (Please refer to the attached image for an email confirming this.)

This pull request introduces several exciting features that enhance the project's usability and aesthetics:

- A new, modal window that shows you all the project's features
- Added animations that appear when the modal opens, providing a more engaging user experience.
- all the project's information comes from a js file and this content is generated by javascript (this reduces the verbose code and makes a more flexible and scalable project)

### CODE REVIEW

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @dhe19 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that does not have Eslint linter checks. remember it helps identify problematic patterns found in your code. please kindly fix that. see how to set it up [here](https://github.com/microverseinc/linters-config/tree/master/javascript)

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: scripts/modal.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+          <div class="modal-actions">
+            <a href="#" class="secondary-button">
+              Open live
+              <img src="./images/svg/icon-github.svg" alt="" />
+            </a>
+            <a href="#" class="secondary-button">
+              See source
+              <img src="./images/svg/icon-open.svg" alt="" />
+            </a>
+          </div>
```
Comment (abdulhamiid) [inline scripts/modal.js]: You have done well implementing the popup window 👍 💯 

## Mobile version
However, there are still issues in your styling we have to fix so as to stick to the design as much as possible according to the [project requirements](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/8_popup.md).  😊🛠

Figma
image

Your design
image

## Desktop version
Figma
image

Your design
image

Please, kindly adjust your design 👍🛠
- The button is `See live` not `Open live`
- The horizontal grey line is missing in your design 👀
- The list of technologies `HTML`, `CSS`, and `JavaScript` should be aligned to the left
- The spacing (margin & padding) on the top, bottom, right, and left between the element

Comment (abdulhamiid) [inline scripts/modal.js]: ### Desktop

Figma
image

Your design
image

- Kindly adjust the spacing (margin & padding) on the top, bottom, right, and left between the element to make your design match the template on Figma 😊💪

**Review by abdulhamiid — CHANGES_REQUESTED** [review]
# Status: Required Changes :recycle:
Hi @dhe19 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! :running_man: :runner: 
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights 🌟

- ✔️ Use basic JavaScript syntax.
- ✔️ Use JavaScript to manipulate DOM elements.
- ✔️ Use JavaScript events.
- ✔️ Understand how to parse a Figma design to create a UI.
- ✔️ Use objects to store and access data.

## Required Changes ♻️
_Please kindly check the comments under the review._

## Optional suggestions 🤷‍♂️
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code._


Cheers, and Happy coding!👏👏👏

## Important Notes 📝
- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: scripts/modal.js (hunk 1 above)
Comment (dhe19) [reply scripts/modal.js]: Hello @abdulhamiid  Currently, I updated this part, I don't understand it pretty well, could check if I could resolve this problem? please. Thank you, have a nice day 😊

**Pull request merged**